azurelinuxagent/common/protocol/wire.py

```diff
@@ -967,24 +1052,36 @@ def send_request_using_appropriate_channel(self, direct_func, host_func):
 
         return ret
 
-    def upload_status_blob(self):
-        ext_conf = self.get_ext_conf()
+    def get_extensions_fast_track_enabled(self):
+        # Eventually we'll need to generalize this method to write all features
+        # but for now we only have one
+        fast_track_enabled = False
+        if conf.get_extensions_fast_track_enabled():
+            if self.hostgaplugin_supports_fast_track():
+                logger.periodic_info(logger.EVERY_DAY, "FastTrack enabled")
```

Comment:
Could we mention the HostGAPlugin here similarly to the message @ line 1064

Reply:
ok